toggle/Toggle.py

```diff
@@ -218,23 +217,22 @@ def loop(self, queue, name):
     """ When a new event comes in, execute it """
     try:
       while self.running:
-        try:
-          update = queue.get(block=True, timeout=1)
-        except Queue.Empty:
+        update = queue.get()
+        if update is None:
```

Comment:
While the logic seems sound, is the change here because queue went from a `JoinableQueue` to a `Queue` instance?
I'm not sure I completely follow the logic change in this case, but I'll admit I'm not super familiar with Python.